[2024-12-18T22:26:53.538+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:26:51.786679+00:00 [queued]>
[2024-12-18T22:26:53.546+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:26:51.786679+00:00 [queued]>
[2024-12-18T22:26:53.547+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-12-18T22:26:53.564+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): env_preparation> on 2024-12-18 22:26:51.786679+00:00
[2024-12-18T22:26:53.570+0000] {standard_task_runner.py:60} INFO - Started process 1503 to run task
[2024-12-18T22:26:53.573+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'env_preparation', 'manual__2024-12-18T22:26:51.786679+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp743075fw']
[2024-12-18T22:26:53.576+0000] {standard_task_runner.py:88} INFO - Job 119: Subtask env_preparation
[2024-12-18T22:26:53.625+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:26:51.786679+00:00 [running]> on host 1d3bd1c7da18
[2024-12-18T22:26:53.702+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='env_preparation' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T22:26:51.786679+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T22:26:51.786679+00:00'
[2024-12-18T22:26:53.720+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-18T22:26:53.730+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=env_preparation, execution_date=20241218T222651, start_date=20241218T222653, end_date=20241218T222653
[2024-12-18T22:26:53.786+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-18T22:26:53.814+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
